### DISCUSSION TITLE
Release 0.21

### DISCUSSION THREAD

**Discussion by MidnightDesign** [infection/infection#1469] [General]
Do you think it makes sense to complete milestone 0.21.0 now and move the open issues to the next milestone? There are already many new features in `master` that would justify a release and most open issues in the milestone are pretty old and don't seem to be worked on.

The discussion around #1215 is also still going on, it seems. And I don't think it needs to be in the next release.

What do you think?

**Comment by maks-rafalko** [comment]
Sure, we will at least complete https://github.com/infection/infection/pull/1465 and ready to go with 0.21.0

I think ETA - approximately next week

Thanks for supporting us on GH Sponsors!
